### DISCUSSION TITLE
Is it possible to keep only the latest notification in the notification drawer?

### DISCUSSION THREAD

**Discussion by momolly1024** [invertase/react-native-firebase#7507] [Q&A]
Due to the notification limit being 24 per app in the notification drawer, 
which function should be used to configure this?

Sorry I am so new in React native and coding.

**Comment by mikehardy** [comment]
device-local notification APIs represent a very challenging space, and this library does not even really attempt to handle them

This library of course may be used to receive cloud messages (via the messaging module) and *sometimes* the underlying SDKs may post a notification in response to that, but for true control over device-local notifications you need a full-fledged notification package like https://notifee.app
